ci-operator/step-registry/ipi/conf/ibmcloud/ipi-conf-ibmcloud-commands.sh

```diff
@@ -51,6 +51,7 @@ credentialsMode: Manual
 platform:
   ibmcloud:
     region: ${REGION}
+    resourceGroupName: ibmcloud-multi-s390x-rg
```

Comment:
From a discussion with Shilpa

I think we need to add an env `RESOURCE_GROUP` to `ci-operator/step-registry/ipi/conf/ibmcloud/ipi-conf-ibmcloud-ref.yaml`

```
  - name: RESOURCE_GROUP
    default: ""
    documentation: |-
      The resource group where the resources are installed in support of the installation.
```

So we can parameterize `resourceGroupName: ibmcloud-multi-s390x-rg` https://github.com/openshift/release/blob/4645cf106424f2f2b6436f1146c2cacbf9e44547/ci-operator/step-registry/ipi/conf/ibmcloud/ipi-conf-ibmcloud-commands.sh#L54

Reply:
agreed, we should parameterize it. for now it was a quick way to use an existing rg, but I will see how we can parameterize it without breaking the existing ibmcloud workflows.

Reply:
I added a preliminary version of the env variable you suggested @prb112 in 672caa130dce4b981f4d4b3ce6190de0efcf8cdb